apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/config.ex

```diff
@@ -514,6 +522,18 @@ defmodule Astarte.DataUpdaterPlant.Config do
               kubernetes_namespace: clustering_kubernetes_namespace!(),
               polling_interval: 10_000
             ]
+          ],
+          vernemq_k8s: [
+            strategy: Elixir.Cluster.Strategy.Kubernetes,
+            config: [
+              mode: :hostname,
+              kubernetes_service_name: "astarte-vernemq",
+              kubernetes_node_basename: "VerneMQ",
+              kubernetes_ip_lookup_mode: :pods,
```

Comment:
Couldn't we use the default value (i.e. `:endpoints`) here?

Reply:
Currently, we cannot.
Libcluster is looking for endpoints with `kubectl get endpoints -l app=astarte-vernemq`.
However the endpoints for VerneMQ do not contain labels: we should change the Astarte Operator to rectify that.

Instead of endpoints, looking for pods already works out of the box.

Reply:
The vernemq endpoint does not have labels (a part for `service.kubernetes.io/headless`, set to `""`) but I dont think it is reliable. If looking for pods, the pods with label `app=astarte-vernemq` uniquely identify vernemq pods in the cluster